apps/studio/components/interfaces/Functions/EdgeFunctionSecrets/EdgeFunctionSecrets.tsx

```diff
@@ -48,11 +48,17 @@ const EdgeFunctionSecrets = () => {
   return (
     <>
       {isLoading && <GenericSkeletonLoader />}
-
       {isError && <AlertError error={error} subject="Failed to retrieve project secrets" />}
 
+      {canUpdateSecrets && (
```

Comment:
Move this inside `isSuccess`. It shows up beneath loading skeleton :).
![Screenshot 2025-02-06 at 15 05 59](https://github.com/user-attachments/assets/349860a2-285d-4674-9ac4-6eec0c655b78)